src/index/am.rs

```diff
@@ -207,8 +207,13 @@ pub unsafe extern "C" fn aminsert(
     let oid = (*index_relation).rd_id;
     let id = get_handle(oid);
     let vector = from_datum(*values.add(0), *is_null.add(0));
+    let multicolumn_data = if (*(*index_relation).rd_index).indnkeyatts == 2 {
+        pgrx::datum::FromDatum::from_datum(*values.add(1), *is_null.add(1)).unwrap()
```

Comment:
We must handle `NULL`s here.

> A related restriction is that an index access method that supports multiple index columns must support indexing null values in columns after the first, because the planner will assume the index can be used for queries that do not restrict these columns.